Formula/lib/libjcat.rb

```diff
@@ -31,13 +31,15 @@ class Libjcat < Formula
   depends_on "json-glib"
   depends_on "nettle"
 
+  uses_from_macos "python"
+
```

Comment:
```suggestion
```
You don't need this.

Reply:
works for me (I think we should probably cleanup the others as well)